src/modules/setupAutorole.ts

```diff
@@ -1,13 +1,43 @@
-import { GuildMember } from 'discord.js';
+import { ChatInputCommandInteraction, GuildMember } from 'discord.js';
 
 import { AUTOROLE_ID } from '../config';
+import { ConfigModel } from '../models/config';
 
 /**
- * Function to setup new guild member autorole
+ * Function to give new guild member autorole
  * @param interaction object which has all information about new member
  */
-export const setupAutorole = async (member: GuildMember) => {
+export const giveAutorole = async (member: GuildMember) => {
+  const config = await ConfigModel.findOne({});
```

Comment:
trzeba bedzie jeszcze informowac uzytkownika jesli awaity sie wysypia to jesli config jest nullem to return cos typu unable to fetch data from mongo

Reply:
To jakbyśmy chcieli informować usera? Na ten moment widzę 2 opcje, albo specjalny kanał na logi od poda, albo wysłać info do usera któremu nadawana jest rola

Reply:
w addNewRoleWithReaction dałem żeby usera informować przy komendzie, to chyba możnaby przy tym zostać póki co - tak naprawdę oba rozwiązania naraz byłyby najlepsze, ale nie robiłbym specjalnego kanału dla bota tylko telemetrię zrobił jakimś frameworkiem

Reply:
ale telemetrię zostawimy na później, póki co zwykla wiadomosc wystarczy